src/modules/patient/models/patient.model.ts

```diff
@@ -186,8 +186,78 @@ export class Patient extends BaseEntity {
         () => Assessment,
         assessment => assessment.patient,
     )
-    assessments: Assessment;
+    assessments: Assessment[];
 
     @OneToMany(() => PatientCaregiver, patientCaregiver => patientCaregiver.patient)
     patientCaregivers: PatientCaregiver[];
 }
+
+@ObjectType()
+export class QuestionnaireChoice {
+    @Field(() => String)
+    _id: string;
```

Comment:
For styling I would recommend, adding at-least one empty line after each field declaration in the class. This has no functional impact, but may make it easier to read through the code later.

Reply:
I agree, if possible we should share a prettier file so formatting will be done automatically and be consistent